src/handler.ts

```diff
@@ -59,6 +59,7 @@ abstract class AbstractFileHandler implements IMessageFilter {
    */
   constructor(manager: IContentsManager) {
     this._manager = manager;
+    document.addEventListener('focus', this._onFocus.bind(this), true);
```

Comment:
you can remove the `.bind` since the member function already closes over `this`.